x-pack/plugins/maps/public/routes/map_page/top_nav_config.tsx

```diff
@@ -195,11 +196,9 @@ export function getTopNavConfig({
               options={tagSelector}
             />
           ) : (
-            <SavedObjectSaveModalDashboard
-              {...saveModalProps}
-              savedObjectsClient={getSavedObjectsClient()}
-              tagOptions={tagSelector}
-            />
+            <PresentationUtilContext>
+              <SavedObjectSaveModalDashboard {...saveModalProps} tagOptions={tagSelector} />
+            </PresentationUtilContext>
```

Comment:
I think you'll want to make the same change we did for visualize here since it's also using `showSaveModal`. Apologies I didn't catch that earlier 😓

Reply:
No sweat! Fixed...